packages/medusa/src/api-v2/admin/price-lists/query-config.ts

```diff
@@ -0,0 +1,63 @@
+import { prefixArrayItems } from "@medusajs/utils"
+import { defaultAdminProductsVariantFields } from "../products/query-config"
+
+export const defaultAdminPriceListFields = [
```

Comment:
Does this match what we had in v1? If we can match it than that'd be great, if not then we'll have to live with the breaking changes :)

Reply:
Yes, the v2 response matches v1 response. The transformation of v2 to v1 happens in the utils function. This is because the data model of pricing between v1 and v2 is vastly different. 

This variable and the ones beneath this is however wrong though. Its meant to showcase a list of attributes that should be present in the response. But in reality, this is a list of remote query fields to query through the price_list entry point. 

I'll look into it a bit more to see how we can clearly depict the 2.